docs/rest_api.md

```diff
@@ -537,11 +537,31 @@ Response body:
 	 "metadata":{"EK":"pk"}
 	}
 
+
+Response body if run by an admin:
```

Comment:
* I might make these examples more symmetric (maybe by adding "if run by a non-admin" to the other one).
* Mention the new fields in the description of the result, not just in an example.